spot_driver/spot_driver/calibrated_reregistered_hand_camera_depth_publisher.py

```diff
@@ -0,0 +1,233 @@
+# Copyreference (c) 2024 Boston Dynamics AI Institute LLC. All references reserved.
+
+"""
+This script exists as it is not currently possible to set the intrinsic and extrinsic
+parameters for the hand cameras through the bosdyn API. Currently, the registered
+depth image is published using extrinsic and intrinsic parameters that are set under the hood. 
+If the updated results from the calibration in spot_wrapper/spot_wrapper/calibration
+can be populated into the underlying bosdyn API, then the default registered depth image
+could be used without this script.
+
+While it is possible to calibrate the gripper camera entirely under the hood, there is no way to
+only set the intrinsic/extrinsic parameters (to preserve control over calibration parameters)
+
+This script assumes you have a calibration generated with /spot_wrapper/spot_wrapper/calibration
+"""
+import argparse
+from typing import Optional
+
+import bdai_ros2_wrappers.process as ros_process
+import cv2
+import numpy as np
+import open3d as o3d
+import yaml
+from bdai_ros2_wrappers.context import wait_for_shutdown
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+
+
+class CalibratedReRegisteredHandCameraDepthPublisher:
```

Comment:
Should this be connected to the launch file?

Reply:
I'm not sure on that front. 

Since it requires a calibration config file, I was hesitant to include it off of the back. I can make a default calibration so it can be included in the launch file, but then I suspect that people might have the overhead of using it without actually calibrating.

Open to suggestions on what's best here!

Reply:
We could use a flag in the launch file, so it isn't on by default but you can start it without needing a separate shell

@tcappellari-bdai @khughes-bdai Thoughts?

Reply:
Yeah, I think a disabled by default option in the launchfile would be useful. Maybe if the user enables it on launch, we enforce having to also pass in the camera calibration file too (and don't launch it if the calibration file isn't included). 
Might make more sense to handle that in another PR though to keep this one from becoming too big